bin/ants

```diff
@@ -156,6 +156,11 @@ def run_ansible(args):
     if args.skip_tags:
         cmd.append('--skip-tags')
         cmd.append(args.skip_tags)
+    if configer.read_config('logstash'):
```

Comment:
This will trigger whether or not `logstash` is in `ansible_callback_whitelist`.

Reply:
You are right. It would be nicer to be consistent with the way we enable it. I will update this tomorrow.